lib/screens/feed_formula_page.dart

```diff
@@ -23,14 +25,31 @@ class FeedFormulaPage extends StatefulWidget {
 }
 
 class _FeedFormulaPageState extends State<FeedFormulaPage> {
-  List<Map<String, dynamic>> fodderItems = [];
-  List<Map<String, dynamic>> concentrateItems = [];
-  final TextEditingController feedWeightController = TextEditingController();
+  List<FeedIngredient> fodderItems = [];
+  List<FeedIngredient> concentrateItems = [];
 
   @override
   void initState() {
     super.initState();
-    feedWeightController.text = '';
+    _loadSavedFeedFormula();
+  }
+
+  void _loadSavedFeedFormula() async {
+    final savedFormula = SharedPrefsService.getFeedFormula();
+    if (savedFormula != null) {
+      setState(() {
```

Comment:
It's worthwhile thinking about error handling here. I am envisioning the next feature being about setting prices and also disabling some feeds (eg. they're not available at that time of year). It could be that some of the changes there would make the saved state illegal (there's an ingredient that's no longer available).

Reply:
Yeah that's what I'm currently working on - managing the feed plan and updated price/ availability